gui/wxpython/psmap/instructions.py

```diff
@@ -86,10 +86,7 @@ def __getitem__(self, id):
 
     def __contains__(self, id):
         """Test if instruction is included"""
-        for each in self.instruction:
-            if each.id == id:
-                return True
-        return False
+        return any(each.id == id for each in self.instruction)
```

Comment:
Would not this be the same?
```suggestion
        return id in self.instruction
```

Reply:
Not exactly, there's a sub-property ".id"

Reply:
Right. Sorry.